src/components/Content/Content.tsx

```diff
@@ -35,6 +46,8 @@ const Content: React.FC<TProps> = (props: TProps) => {
         </ul>
       </aside>
       <main className={styles.contentMain}>
+        {error && <Errors>{error}</Errors>}
+        {!props.movies.length && <Errors>{noResults}</Errors>}
```

Comment:
`!props.movies.length` movies.length'in olmaması anlamına gelir. undefined,null gibi durumlarda da buraya düşer. Aradığın şey `props.movies.length === 0` durumu